css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
+	width: 100%;
+	height: 640px;
+}
+
+.firstRectangle{
+	background-image: url('../img/feed.jpg');
+	background-blend-mode: overlay;
+}
+
+.whiteRectangle{
+	width: inherit;
+	height: inherit;
+	background-color: #CBF6FF;
+	opacity: 0.3;
+	position: absolute;
+}
+
+.logInBox{
+	width: 375px;
+	height: 400px;
+	margin-left: 400px;
+	background-color: #F5F5F5;
+	opacity: 1!important;
+	position: absolute;
+	margin-top: 140px;
+	box-shadow: 0px 0px 8px black;
```

Comment:
You're not wrong, but I'm not right either. :P What I was trying to get at is the fact that `0` is `0`, regardless of unit of measurement. Zero pixels or zero percent of anything is still equal to zero. Not to say that you _need_ to remove the measurement units (feel free to keep them in if you feel it's worth it), but just making sure you're aware that you have the option to leave them out for `0`, specifically.